.README.html

```diff
@@ -209,6 +209,8 @@ <h1 class="toc-title">Contents</h1>
 id="toc-ad_integration_sssd_custom_settings">ad_integration_sssd_custom_settings</a></li>
 <li><a href="#ad_integration_preserve_authselect_profile"
 id="toc-ad_integration_preserve_authselect_profile">ad_integration_preserve_authselect_profile</a></li>
+<li><a href="#ad_integration_manage_packages"
+id="toc-aad_integration_manage_packages">ad_integration_manage_packages</a></li>
```

Comment:
Please move these changes to README.md. .README.html is generated automatically from README.md